CODEOWNERS

```diff
@@ -23,3 +23,5 @@
 
 /definitions/npm/sequelize_*/**/* @jedwards1211
 
+/definitions/npm/kafkajs_*/**/* @edoardo-bluframe
+/definitions/npm/react-helmet_v6*/**/* @edoardo-bluframe
```

Comment:
Check out [this](https://github.com/flow-typed/flow-typed/blob/main/CONTRIBUTING.md#owning-definitions) to add code owners in the respective definition instead for the service to do it's thing